src/extension.ts

```diff
@@ -24,9 +25,45 @@ export async function activate(context: vscode.ExtensionContext) {
                     await activateConfigurePipeline();
             },
             TelemetryKeys.ExtensionActivationDuration);
-    });
+	});
+	
+    // The YAML language server is implemented in node
```

Comment:
Can this language client and server configuration fail? If so, we should move it inside try-catch and suppress (and send telemetry) any failure. This is an additional feature.

Reply:
It might fail if some changes are made on the server-side. As a precaution, I'll add a try-catch block and log the telemetry.

Reply:
@JA-04 you can also move the code to a separate function just to keep activate() short and easy to read.

Reply:
Okay, will create a new function for this.